languages/english.php

```diff
@@ -116,3 +116,7 @@
 $lang["WITHPATH"] = "With Path:";
 $lang["FILEDELETED"] = "File deleted";
 $lang["FILEADDED"] = "New file";
+
+$lang['SEARCH'] = "Search";
+$lang['SEARCH_PLACEHOLDER'] = "Search for file name here";
```

Comment:
Files only? Won't it match directories?

Reply:
It matches directories as well. It is just some text written for help. Hoping one of you would let me know what to write here. :)

Reply:
Make it agnostic: "Search for items here"

Reply:
Changed the value for key 'SEARCH_PLACEHOLDER' to "Search for items here."